repo/db/messages.go

```diff
@@ -21,15 +22,15 @@ func NewMessageStore(db *sql.DB, lock *sync.Mutex) repo.MessageStore {
 }
 
 // Put will insert a record into the messages
-func (o *MessagesDB) Put(messageID, orderID string, mType pb.Message_MessageType, peerID string, msg repo.Message) error {
+func (o *MessagesDB) Put(messageID, orderID string, mType pb.Message_MessageType, peerID string, msg repo.Message, rErr error, receivedAt int) error {
```

Comment:
`receivedAt` should be int64 so we don't unintentionally allow 32-bit values on 32-bit arch machines.